### PR TITLE
Update the sku items order

### PR DESCRIPTION
**Pull request by vitoria**
#### What is the purpose of this pull request?
Update the sku items to be sorted by each one's price.

#### What problem is this solving?
The sku items weren't sorted.

#### How should this be manually tested?
http://skusorted--storecomponents.myvtex.com/

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [X] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Add `SkuSelector` and `Share` components.
 
+### Updated
```
Comment (andreldsa) [inline CHANGELOG.md]: Changed*

**Review by andreldsa — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vitoria) [reply CHANGELOG.md]: Done!

**Review by andreldsa — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 - Update props passed to `ShippingSimulator`.
+- Sku items sorted by the each one's price.
```
Comment (salesfelipe) [inline CHANGELOG.md]: Sku items sorted by price

File: react/ProductDetails.js [hunk 3]
```diff
@@ -26,9 +26,19 @@ class ProductDetails extends Component {
   }
 
   handleSkuChange = skuIndex => {
-    this.setState({skuIndex})
+    this.setState({ skuIndex })
   }
 
+  compareSku = (item1, item2) => {
```
Comment (salesfelipe) [inline react/ProductDetails.js]: This sort function shouldn't be inside the component

**Review by salesfelipe — CHANGES_REQUESTED** [review]

**Review by salesfelipe — APPROVED** [review]

**Pull request merged**